LOH/1.0.0/Dockerfile

```diff
@@ -1,25 +1,36 @@
 FROM ubuntu:20.04
 LABEL maintainer="Saksham Phul (phuls@chop.edu)"
-LABEL description="Install gcc, bcftools with plugins, python 3.10, pandas, pysam"
+LABEL description="Install gcc, bcftools with plugins, python 3.10, pandas, g++, cmake, bam-readcount for LOH app"
 
 WORKDIR home/
 
+ENV BCFTOOLS_VERSION 1.16
+ENV BAM_READCOUNT_VERSION 1.0.1
+
 RUN apt update && apt upgrade -y && \
         apt install -y software-properties-common -y && add-apt-repository ppa:deadsnakes/ppa
 
 #apt install cannot be combined, need to link repo to OS to find required packages
-RUN apt install -y curl gcc make liblzma-dev zlib1g-dev libbz2-dev wget python3.10 python3.10-distutils bzip2 && \
+RUN apt install -y curl gcc make liblzma-dev zlib1g-dev libbz2-dev wget python3.10 python3.10-distutils bzip2 g++ cmake git && \
 	curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10  && \
 	update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.10 1 && \
-        wget https://github.com/samtools/bcftools/releases/download/1.16/bcftools-1.16.tar.bz2 && \
-        tar -axf bcftools-1.16.tar.bz2 && \
-        rm bcftools-1.16.tar.bz2 && \
-	cd bcftools-1.16 && \
+        wget https://github.com/samtools/bcftools/releases/download/${BCFTOOLS_VERSION}/bcftools-${BCFTOOLS_VERSION}.tar.bz2 && \
```

Comment:
Very much improved, thanks! Perhaps one more thing that I liked that @dmiller15 pointed out to me. You can shrink the download tar ball + untar + rm tar ball with piped curl command. See https://github.com/d3b-center/bixtools/blob/58855b182e92565c9bd7a5aa4c051fad3b20d967/FusionAnnotator/0.1.1/Dockerfile#L36 for an example.